app/containers/MapContainer/utils.js

```diff
@@ -57,6 +57,105 @@ export const valueOfCircle = (radius, range, config) => {
   return scale(radius);
 };
 
+export const filterFeatureMaxZoom = (feature, zoom) => {
```

Comment:
could be renamed to filterFeaturesByZoom

could be made more generic by passing the property name, i.e. `propertyMaxZoom` as an argument